src/plugins/discover/public/application/components/discover_grid/discover_grid.tsx

```diff
@@ -131,6 +136,14 @@ export interface DiscoverGridProps {
    * How the data is fetched
    */
   useNewFieldsApi: boolean;
+  /**
+   * Manage pagination control
+   */
+  isPaginationEnabled?: boolean;
+  /**
+   * List of used control columns
+   */
+  controlColumnIds?: string[];
```

Comment:
I'd recommend adding info which are available to doc , so it's visible on the first sight